### DISCUSSION TITLE
Does "Refresh" button fetches new entries or just reloads view without full page reload?

### DISCUSSION THREAD

**Discussion by Jorenar** [Athou/commafeed#1035] [Q&A] [answered]
In relation to issue #1034

**Comment by Athou** [comment, marked as answer]
It currently only reloads the view. I now realize that I forgot to port this feature from CommaFeed 2.x :)

![image](https://user-images.githubusercontent.com/1256795/212460601-e35a0c0f-27b9-49d0-9f95-3a99701260d0.png)

**Reply by Jorenar** [reply]
I've added #1036 then
